docs/en/styling-and-theming/supplemental.md

```diff
@@ -419,32 +268,38 @@ Extending the above example:
 }
 ```
 
-> src/widgets/MyThemeableTsxWidget.tsx
+> src/widgets/MyThemeableWidget.tsx
 
-```tsx
-import WidgetBase from '@dojo/framework/core/WidgetBase';
-import ThemedMixin, { theme } from '@dojo/framework/core/mixins/Themed';
-import { tsx } from '@dojo/framework/core/vdom';
+```ts
+import { create, tsx } from '@dojo/framework/core/vdom';
+import theme from '@dojo/framework/core/middleware/theme';
 
-import * as commonCss from '../styles/MyThemeCommonStyles.m.css';
 import * as css from '../styles/MyThemeableWidget.m.css';
+import * as commonCss from '../styles/MyThemeCommonStyles.m.css';
 
-@theme(commonCss)
-@theme(css)
-export default class MyThemedTsxWidget extends ThemedMixin(WidgetBase) {
-	protected render() {
-		return (
-			<div classes={[this.theme(css.root), this.theme(commonCss.commonBase), css.myWidgetExtraClass]}>
-				Hello from a themed Dojo TSX widget!
-			</div>
-		);
-	}
-}
+const factory = create({ theme });
+
+export default factory(function MyThemeableWidget({ middleware: { theme } }) {
+	const { root } = theme.classes(css);
+	const { commonBase } = theme.classes(commonCss);
+	return (
+		<div
+			classes={[
+				/* requirement 2: */
```

Comment:
req 3?

Reply:
Think this one is still incorrect? Although might be best just removing the comment unless we want to add a req #2 comment to the `theme.classes()` calls above.